### PR TITLE
Added minikube example

### PR DESCRIPTION
**Pull request by mrsabath**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +106,71 @@ Then recreate a new, empty namespace:
...
+Otherwise, follow the simple steps below to create a Vault instance, as a pod and
+service, deployed in `trusted-identity` namespace in your cluster.
+
+```console
+kk create -f examples/vault-plugin/vault.yaml
+service/tsi-vault created
+deployment.apps/tsi-vault created
+```
+
+#### Obtain remote access to Vault service
```
Comment (lumjjb) [inline README.md]: Suggestion: we can do a collapsable for this depending on minikube vs IKS something like this https://github.com/cncf/sig-security#sig-members

File: examples/vault-plugin/demo.registerJSS.sh [hunk 2]
```diff
@@ -18,7 +18,7 @@ HELPMEHELPME
 }
 kk="kubectl -n trusted-identity"
 
-# this function registers individualx
+# this function registers individual node
```
Comment (lumjjb) [inline examples/vault-plugin/demo.registerJSS.sh]: nit : individual nodes

Comment (mrsabath) [reply examples/vault-plugin/demo.registerJSS.sh]: Actually, it's a single node, one by one

File: README.md [hunk 3]
```diff
@@ -17,7 +17,8 @@ physical hardware. Secrets are released to the application based on this identit
 ## Table of Contents
 - [Installation](./README.md#installation)
   - [Prerequisites](./README.md#prerequisites)
-  - [Setup Cluster](./README.md#setup-cluster)
+  - [Setup Vault](.README.md#setup-vault)
```
[the remark this answers is not in the document]
Comment (mrsabath) [reply README.md]: thx

File: MAINTAINERS.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
+# Maintainers Guide
```
Comment (lumjjb) [inline MAINTAINERS.md]: I think this document should be `CONTRIBUTING.md` and `MAINTAINERS` should be a separate file.

File: MAINTAINERS.md [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+
+This guide is intended for maintainers - anybody with commit access to Trusted
+Service Identity repository.
+
+[Maintainers List](./MAINTAINERS.md#maintaintainers-list)
```
Comment (lumjjb) [inline MAINTAINERS.md]: Typo in anchor link

File: README.md [hunk 6]
```diff
@@ -24,10 +24,8 @@ physical hardware. Secrets are released to the application based on this identit
   - [Cleanup](./README.md#cleanup)
 - [Usage (demo)](examples/README.md)
 - [Integrating TSI with your Application (Vault example)](examples/README-AppDeveloperVault.md)
-- [Integrating TSI with your Application (Key Store example)](examples/README-AppDeveloperKeyServer.md)
 - [Contributing (TSI Development)](./CONTRIBUTING.md)
-- Extras
-  - [Automated Vault Certificate Management](./CONTRIBUTING.md#automate-vault-certificates)
+- [Maintainers List](./MAINTAINERS.md#maintaintainers-list)
```
Comment (lumjjb) [inline README.md]: typo in anchor link

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -3,6 +3,18 @@
 If you are interested in contributing and developing the Trusted Service Identity
 follow the steps below.
 
+## Reporting security issues
```
Comment (lumjjb) [inline CONTRIBUTING.md]: I think this should be in the README

**Pull request merged**